.gitignore

```diff
@@ -6,3 +6,4 @@ crosstool-ng
 aesd-build
 .vscode/
 test.sh.log
+actions-runner
```

Comment:
@Suraj-Ajjampur you should also add `*.swp` here and avoid checking in the .swp file like the CMakeLists.txt.swp above.